src/ncp/ncp_base.cpp

```diff
@@ -885,7 +888,7 @@ ThreadError NcpBase::SendPropertyUpdate(uint8_t header, uint8_t command, spinel_
 
 ThreadError NcpBase::OutboundFrameFeedVPacked(const char *pack_format, va_list args)
 {
-    uint8_t buf[64];
+    uint8_t buf[256];
```

Comment:
Adjusting this buffer size shouldn't be necessary. For larger buffers you should be able to call `OutboundFrameFeedData()` directly.